### PR TITLE
Properly display error message in whitelist function

### PR DESCRIPTION
**Pull request by DRSchlaubi**
Fixes #3

### CODE REVIEW

File: kordx-commands-runtime/src/main/kotlin/com/gitlab/kordlib/kordx/commands/argument/text/Extensions.kt [hunk 1]
```diff
@@ -18,10 +19,10 @@ fun <CONTEXT> Argument<String, CONTEXT>.whitelist(
             when {
                 ignoreCase -> when {
                     whitelist.any { word -> word.equals(it, true) } -> FilterResult.Pass
-                    else -> FilterResult.Fail("expected one of $whitelist (not case sensitive) but got $it")
+                    else -> FilterResult.Fail("expected one of ${whitelist.joinToString(", ")} (not case sensitive) but got $it")
                 }
                 it in whitelist -> FilterResult.Pass
-                else -> FilterResult.Fail("expected one of $whitelist (case sensitive) but got $it")
+                else -> FilterResult.Fail("expected one of ${whitelist.joinToString(", ")} (case sensitive) but got $it")
```
Comment (HopeBaron) [inline kordx-commands-runtime/src/main/kotlin/com/gitlab/kordlib/kordx/commands/argument/text/Extensions.kt]: Please use `contentToString` instead for a list-like string representation.

**Review by HopeBaron — CHANGES_REQUESTED** [review]

File: kordx-commands-runtime/src/main/kotlin/com/gitlab/kordlib/kordx/commands/argument/text/Extensions.kt (hunk 1 above)
Comment (DRSchlaubi) [reply kordx-commands-runtime/src/main/kotlin/com/gitlab/kordlib/kordx/commands/argument/text/Extensions.kt]: Wasn't sure what looks better

**Review by HopeBaron — COMMENTED** [review]
The `options` variable should be `private`.
Thanks for contributing to Kord.

**Comment by DRSchlaubi** [comment]
Oh I made that public, I guess that's why they added that explicit API mode

**Comment by BartArys** [comment]
The visibility doesn't really matter since the implementing class is effectively private. Still, good coding conventions and all that.

**Pull request closed without merging**